### PR TITLE
fix(audio): error handling for missing audio output devices

### PR DESCRIPTION
**Pull request by nathan-power**
**What this PR does** 📖
- Fixes the dropdown for Video Input not opening when user clicks on it in Safari.

**Which issue(s) this PR fixes** 🔨
AP-1914

**Special notes for reviewers** 🗒️
- The "Video Input" dropdown problem was caused by an error spawning from missing audio output devices, as Safari does not allow setting “audio out”, which resulted in the function that set up the "Video Input" dropdown being exited before options could be set.

**Additional comments** 🎤

### CODE REVIEW

**Review by jasonwoodland — CHANGES_REQUESTED** [review]

File: components/views/settings/pages/audio/index.vue [hunk 1]
```diff
@@ -268,10 +268,10 @@ export default Vue.extend({
 
         // Setting defaults on mount if one isn't already present in local storage
         if (!this.settings.audioInput) {
-          this.isAudioInput = permissionsObject.devices.audioIn[0].value // chrome, ffx, and safari all support the audioIn object
+          this.isAudioInput = permissionsObject.devices.audioIn[0]?.value // chrome, ffx, and safari all support the audioIn object
```
[the remark this answers is not in the document]
Comment (nathan-power) [reply components/views/settings/pages/audio/index.vue:271]: All good, will get on it now

File: components/views/settings/pages/audio/index.vue [hunk 2]
```diff
@@ -247,6 +247,7 @@ export default Vue.extend({
      */
     async setupDefaults() {
       const permissionsObject: any = await this.getUserPermissions()
+      console.log('permissionsObject', permissionsObject)
```
Comment (josephmcg) [inline components/views/settings/pages/audio/index.vue:250]: console log snuck in

Comment (nathan-power) [reply components/views/settings/pages/audio/index.vue:250]: Well that's no good

**Comment by josephmcg** [comment]
this one can be tested properly, doesn't rely on any missing functionality

**Pull request merged**